maskpostgresdata/management/commands/dump_masked_data.py

```diff
@@ -27,6 +27,26 @@ def add_arguments(self, parser):
     def update_auth_user(self, queryset):
         queryset.update(password=make_password("password"))
 
+    def reset_sequences(self, cursor):
+        """
+        Get all the sequances from the database and the last values for it
+        """
+        # Get all sequances
+        cursor.execute("SELECT * FROM information_schema.sequences;")
```

Comment:
Maybe change this to just `SELECT sequence_name`, and tweak the code further down?

Optional!